drivers/filesystems/fastfat/cleanup.c

```diff
@@ -29,14 +30,14 @@ VfatCleanupFile(
     BOOLEAN IsVolume;
     PDEVICE_EXTENSION DeviceExt = IrpContext->DeviceExt;
     PFILE_OBJECT FileObject = IrpContext->FileObject;
-
+    BOOLEAN Deleted = FALSE;
     DPRINT("VfatCleanupFile(DeviceExt %p, FileObject %p)\n",
```

Comment:
Maybe just add an extra newline before the dprint to separate that list of variables with the rest of the code.